libs/langgraph/src/pregel/retry.ts

```diff
@@ -184,18 +147,21 @@ async function _runWithRetry(
       await new Promise((resolve) => setTimeout(resolve, intervalWithJitter));
       // log the retry
       const errorName =
-        error.name ??
+        (error as Error).name ??
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (error.constructor as any).unminifiable_name ??
-        error.constructor.name;
+        ((error as Error).constructor as any).unminifiable_name ??
+        (error as Error).constructor.name;
       console.log(
-        `Retrying task "${pregelTask.name}" after ${interval.toFixed(
+        `Retrying task "${String(pregelTask.name)}" after ${interval.toFixed(
```

Comment:
this doesn't do anything, right? string interpolation does calls toString on things anyway

Reply:
Ah right, it's necessary because the type of `PregelExecutableTask.name` is `string | Symbol`.